public/locales/en/common.json

```diff
@@ -122,8 +122,9 @@
   "footer": {
     "links": {
       "disclaimer": "Disclaimer",
-      "docs": "Documentation",
-      "gov": "Governance forum",
+      "docs": "Docs",
```

Comment:
<img width="622" alt="image" src="https://user-images.githubusercontent.com/78362532/163481019-1a1198c5-4ca4-4ecc-a6ce-d43a62fc5da7.png">
These are the texts in the current design, are these changes approved by copy/design team? @FoxTheSin

Reply:
Approved by Evan after fox asking, resolved